samples/relocate_events.py

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env python
+"""
+Written by Nathan Prziborowski
+Github: https://github.com/prziborowski
+
+This code is released under the terms of the Apache 2
+http://www.apache.org/licenses/LICENSE-2.0.html
+
+Simple example for getting the vMotion/relocate events of a VM.
+There are additional filters for time that I didn't include as I
+didn't want to add time parsing complications.
+
+"""
+import re
+import sys
+from pyVmomi import vim
+from pyVim.connect import SmartConnect
+from pyVim.task import WaitForTask
+from tools import cli
+
+__author__ = 'prziborowski'
+
+
+def setup_args():
+    parser = cli.build_arg_parser()
+    parser.add_argument('-n', '--name',
+                        help='Name of the VM to test CD-rom on')
+    parser.add_argument('-d', '--datacenter',
+                        help='Name of datacenter to search on. '
+                             'Defaults to first.')
+    parser.add_argument('--filterUsers',
+                        help="Comma-separated list of users to filter on")
+    parser.add_argument('--filterSystemUser', action='store_true',
+                        help="Filter system user, defaults to false.")
+    return cli.prompt_for_password(parser.parse_args())
+
+
+def main():
+    args = setup_args()
+    si = SmartConnect(host=args.host, user=args.user, pwd=args.password)
```

Comment:
You can use SmartConnectNoSSL in order to support 2.7.9+ and baypass the cert verification.